### PR TITLE
chore: aria rebranding - cmdlet content

### PR DESCRIPTION
**Pull request by tenthirtyam**
### Summary

- Updated cmdlet descriptions and examples for Aria Suite, formerly known as vRealize Suite, products to use the new Aria names. 
- Updated `CHANGELOG.md`.
- Updated module version to v2.7.0.1008.

**Note:** No changes to the names of the cmdlets or variables. Not in scope.

### Type

- [ ] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [x] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

- [ ] Tests have been completed.
- [x] Documentation has been added or updated.

### CODE REVIEW

File: PowerValidatedSolutions.psm1 [hunk 1]
```diff
@@ -2495,7 +2495,7 @@ Function Get-NSXLBDetails {
         # Retrieve WSA VIP
```
[the remark this answers is not in the document]
Comment (tenthirtyam) [reply PowerValidatedSolutions.psm1:2495]: Resolved in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/eff0306cc80dc72a3f0c859e05941f8d4f7fc6a7.

File: PowerValidatedSolutions.psm1 [hunk 2]
```diff
@@ -7536,18 +7535,18 @@ Function Undo-vRLIDeployment {
...
-                                            Write-Error "Removal of vRealize Log Insight from vRealize Suite Lifecyle Manager ($($vcfVrslcmDetails.fqdn)): POST_VALIDATION_FAILED"
+                                            Write-Error "Removal of Aria Operations for Logs from Aria Suite Lifecycle ($($vcfVrslcmDetails.fqdn)): POST_VALIDATION_FAILED"
                                         }
                                     } else {
-                                        Write-Output "Removal request of vRealize Log Insight Submitted Successfully (Request Ref: $($newRequest.requestId))"
+                                        Write-Output "Removal request of Aria Operations for Logs Submitted Successfully (Request Ref: $($newRequest.requestId))"
                                     }
                                 } else {
-                                    Write-Error "Removel request of vRealize Log Insight failed, check the vRealize Suite Lifecycle Manager UI: POST_VALIDATION_FAILED"
+                                    Write-Error "Removel request of Aria Operations for Logs failed, check the Aria Suite Lifecycle UI: POST_VALIDATION_FAILED"
```
Comment (bhumitra) [inline PowerValidatedSolutions.psm1:7546]:
```suggestion
                                    Write-Error "Removal request of Aria Operations for Logs failed, check the Aria Suite Lifecycle UI: POST_VALIDATION_FAILED"
```

**Review by bhumitra — CHANGES_REQUESTED** [review]
Just one more. This is impressive what you have done! Heck of a task!

File: PowerValidatedSolutions.psm1 (hunk 2 above)
Comment (tenthirtyam) [reply PowerValidatedSolutions.psm1:7546]: Nice catch! Resolved in force push to https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/80a44deb6323815e007842449672e8aaacdce83c.

**Pull request merged**